src/domain/occurrence/__tests__/OccurrenceDetailsPage.test.tsx

```diff
@@ -0,0 +1,165 @@
+// import { axe, toHaveNoViolations } from 'jest-axe';
+import React from 'react';
+
+import {
+  EventDocument,
+  OccurrenceDocument,
+  PlaceDocument,
+  VenueDocument,
+} from '../../../generated/graphql';
+import {
+  renderWithRoute,
+  screen,
+  userEvent,
+  waitFor,
+} from '../../../utils/testUtils';
+import { ROUTES } from '../../app/routes/constants';
+import OccurrenceDetailsPage from '../OccurrenceDetailsPage';
+import {
+  eventResult,
+  occurrenceResult,
+  placeResult,
+  venueResult,
+} from './mocks';
+
+// expect.extend(toHaveNoViolations);
```

Comment:
You can import 'jest-axe/extend-expect' on src/setupTests.ts so you don't have to extend expect object for each test file. e.g. https://github.com/City-of-Helsinki/events-helsinki-ui/blob/develop/src/setupTests.ts

Reply:
yeah I will make that change :)